include/Model/Languages.php

```diff
@@ -27,8 +27,12 @@ class Languages {
 	public const LOCALE_PATTERN = '^' . self::INNER_LOCALE_PATTERN . '$';
 	public const SLUG_PATTERN   = '^' . self::INNER_SLUG_PATTERN . '$';
 
-	public const TRANSIENT_NAME = 'pll_languages_list';
-	private const CACHE_KEY     = 'languages';
+	public const TRANSIENT_NAME      = 'pll_languages_list';
+	public const TRANSFORMATIVE_ARGS = array(
+		'fields' => false,
+	);
```

Comment:
I find it to be more confusing with this constant than using `unset( $args['fields'])` or something.
I don't see any case where this list will grow, this is more a backward compatibility with legacy code.
If you really want a constant, I suggest to rename to something like `FIELDS_LIST_ARG` or something.

Reply:
The aim is for the proxies to not know about `$args['fields']`. So you prefer `public const FIELDS_LIST_ARG = 'fields';`?